apps/browser/settings/declarativeloginmodel.cpp

```diff
@@ -127,6 +133,11 @@ void DeclarativeLoginModel::setLoginList(const QVariantList &data)
     }
     endResetModel();
     emit countChanged();
+
+    if (!m_populated) {
```

Comment:
I hope that now we do not assume the possibility of re-updating the model and we do not need to update "popuplated" more than once

Reply:
It's not clear from this PR alone, but I've make this assumption in other parts of the code too, given the model gets recreated every time you enter the page.

In the unit test PR for the login page (under review and to be applied alongside this) the observer gets deregistered so that it won't update (see line 122 of declarativeloginmode.cpp here https://github.com/sailfishos/sailfish-browser/pull/839).